invenio_requests/assets/semantic-ui/translations/invenio_requests/scripts/initCatalog.js

```diff
@@ -0,0 +1,20 @@
+// This file is part of React-Invenio-Deposit
+// Copyright (C) 2021 Graz University of Technology.
```

Comment:
```suggestion
// This file is part of Invenio-Requests
// Copyright (C) 2021 Graz University of Technology.
// Copyright (C) 2022 CERN.
```

Same question as above.